Cargo.toml

```diff
@@ -1,19 +1,5 @@
 [workspace]
-
-members = [
-    "application/comit_node",
-    "application/ledger_query_service",
-    "vendor/bam",
-    "vendor/bitcoin_rpc_test_helpers",
-    "vendor/bitcoin_support",
-    "vendor/bitcoin_witness",
-    "vendor/ethereum_support",
-    "vendor/key_gen",
-    "vendor/secp256k1_support",
-    "vendor/tc_bitcoincore_client",
-    "vendor/tc_web3_client",
-    "vendor/noise_handshake"
-]
+members = ["application/comit_node", "application/ledger_query_service", "vendor/bam", "vendor/bitcoin_rpc_test_helpers", "vendor/bitcoin_support", "vendor/bitcoin_witness", "vendor/ethereum_support", "vendor/key_gen", "vendor/secp256k1_support", "vendor/tc_bitcoincore_client", "vendor/tc_web3_client", "vendor/noise_handshake"]
```

Comment:
:x: This is actually a no-go for me.

Can we open an issue that this is formatted differently?

Reply:
Ah yeah, I remember this. I think I tweeted with the creator about it. Please open an issue and see if we can configure it :)
OR, we don't apply the formmating on the meta cargo.toml

Reply:
Ah yeah, I remember this. I think I tweeted with the creator about it. Please open an issue and see if we can configure it :)
OR, we don't apply the formmating on the meta cargo.toml